### ISSUE TITLE
Console component needed 

### ISSUE THREAD

**Issue by lwrigh** [patternfly/patternfly-next#1793]
We have a console component in RHV that allows the user to view their VM in browser. A PF4 component for this will be necessary for a PF4 migration in RHV. 

Current console page: 
![52848569-5ca37f00-310f-11e9-9c71-8e236645ab01](https://user-images.githubusercontent.com/20227374/57137719-b7e42580-6d7e-11e9-96d0-c3fc8dc460ef.png)

**Comment by rachael-phillips** [comment]
Thanks for filing @lwrigh ! @LHinson fyi

**Comment by rachael-phillips** [comment]
Hi @lwrigh I'm following up here on this issue. Do you have any timeline or priority needs for this component?

**Comment by lwrigh** [comment]
@rachael-phillips The PF3 one will be merged soon so we probably won't need a PF4 one until the RHV 4.4 release next year at the absolute earliest.

**Comment by rachael-phillips** [comment]
Thank you for letting me know!

**Comment by mcarrano** [comment]
@lwrigh is this still a need, and if so, exactly what are the requirements to apply to a PatternFly component.

**Comment by lwrigh** [comment]
@mcarrano Yes it's still a need for transitioning the VM portal in RHV over to PF4 fully. The requirements based off of the current implementation would be a console component that supports a full screen mode, a toggle to go between console types, an empty state if the console experiences an error, and a place to provide the user with instructions for using the console. 

Here's a screenshot of the current console implementation: 
<img width="1440" alt="Screen Shot 2019-12-16 at 3 19 44 PM" src="https://user-images.githubusercontent.com/20227374/70940072-9129e500-2017-11ea-9f88-2647c6683c06.png">

**Comment by mcarrano** [comment]
@tlabaj @mcoker can you take a look at this? Looks like this was offered as a React component in PF3, but for PF4 would we consider this a component or more like a demo?

**Comment by lizsurette** [comment]
Based on review today in the PF meeting...we need to consider if this makes sense to be a component for PatternFly 4. Should it simply be a component built by a product like RHV or CNV that takes the Toolbar and then uses the empty state, serial terminal output, or console connection output? 

I think it makes sense to pull in some product developers to the conversation.

FYI @lwrigh @mcarrano

**Comment by mcarrano** [comment]
Agree @lizsurette . My thinking is that from a PF perspective, this may be just a container where someone can drop anything - a console, terminal window, or even an editor. The ability to make this go full screen is an important requirement and would be a useful add to PF IMO. Going to pull this into the PF design backlog for further consideration. @rachael-phillips @LHinson should evaluate for roadmap prioritization.
